plugin/sdk/bus/messages.py

```
@@ -0,0 +1,272 @@
+                time.sleep(check_interval)
+                continue
+            if response.get("error"):
+                raise RuntimeError(str(response.get("error")))
+
+            result = response.get("result")
+            if isinstance(result, dict):
+                return bool(result.get("deleted"))
+            return False
+
+        _ = state.get_plugin_response(req_id)
+        raise TimeoutError(f"MESSAGE_DEL timed out after {timeout}s")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)